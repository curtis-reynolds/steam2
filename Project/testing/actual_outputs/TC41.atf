05_buystdwcredit___sellstd_________000100.00
00_admin___________AA_000000.00
