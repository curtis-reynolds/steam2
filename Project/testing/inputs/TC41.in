login
admin
refund
sellstd
buystdwcredit
100
