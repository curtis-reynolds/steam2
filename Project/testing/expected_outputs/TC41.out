Login successful.
Refund successful. Transferred 100 credits from buystdwcredit to sellstd.
